### PR TITLE
Mosaic 2x2 banners

### PR DESCRIPTION
**Pull request by moontrip**
This will resolve https://github.com/VEuPathDB/web-eda/issues/1383 and https://github.com/VEuPathDB/web-eda/issues/1574. Screenshots are attached in the following:

a) showing top and stats table banners at once (unfolded)
mosaic-2x2-banner00

b) showing top banner content after folded
mosaic-2x2-banner01

c) showing stats table banner content after folded: no facet variable
mosaic-2x2-banner02

d) a case with a facet variable (unfolded)
mosaic-2x2-banner03

### CODE REVIEW

File: src/lib/core/components/visualizations/implementations/MosaicVisualization.tsx [hunk 1]
```diff
@@ -736,272 +771,722 @@ function MosaicViz(props: Props<Options>) {
...
+}) {
+  return props != null ? (
+    <>
+      {props.facetVariableDetails != null ? (
+        ''
+      ) : (
+        <div
+          style={{
+            marginLeft: '0.9em',
+            width: '750px',
```
Comment (jernestmyers) [inline src/lib/core/components/visualizations/implementations/MosaicVisualization.tsx:1280]: I wonder if we want the width to increase when faceted?

**Comment by moontrip** [comment]
Thank you for your suggestions concerning the width of stats' table banner for the case of facet variable, @jernestmyers @asizemore Actually I was gauging whether it should be full width or not so your suggestion convinced me of making a decision. I will update it 👍

**Comment by moontrip** [comment]
> 
> Also, I think we should open a follow-up ticket in `CoreUI` to change the cursor of the `collapsibleIcon` to be `pointer`, which would better indicate that it's a clickable button.
> 
> [Here's the relevant code](https://github.com/VEuPathDB/CoreUI/blob/e9686d345ec809e74843e65be91491bc9fe3b55e/src/components/banners/Banner.tsx#L240-L255) in the `Banner` component.

Definitely I like the idea @jernestmyers: I was thinking the case before but didn't have time to do that. Perhaps you can easily handle it at a follow-up ticket in the near future 😄

**Comment by jernestmyers** [comment]
> Definitely I like the idea @jernestmyers: I was thinking the case before but didn't have time to do that. Perhaps you can easily handle it at a follow-up ticket in the near future 😄

Yes, I'll open the issue now and try to get to it soon!

**Comment by moontrip** [comment]
@jernestmyers @asizemore As suggested, I made a full width of stats table banner when a facet variable exists. Here is a screenshot:

mosaic-2x2-banner04

**Pull request merged**